addons/mail/static/src/messaging/component/messaging_menu/messaging_menu.js

```diff
@@ -52,11 +52,18 @@ class MessagingMenu extends Component {
     // Public
     //--------------------------------------------------------------------------
 
+    /**
+     * @returns {mail.messaging.entity.Discuss}
+     */
+    get discuss() {
```

Comment:
would not use a getter here

Reply:
I would keep it:
- verbosity: the existence check of messaging in the getter makes the template too hard to read
- consistency: this was the only use of discuss in a template without the getter
- future: I would tend to prefer to hide all singletons from components if possible. I don't like seeing "env" in a template in general.